dart/test/monster_test_my_game.example2_generated.dart

```diff
@@ -26,6 +27,37 @@ class Monster {
   String toString() {
     return 'Monster{}';
   }
+
+  MonsterT unPack() => MonsterT();
```

Comment:
here and elsewhere: I'd expect this to be `unpack`

Reply:
OK, renamed.